vkconfig/dlgprofileeditor.cpp

```diff
@@ -391,7 +390,7 @@ void dlgProfileEditor::currentLayerChanged(QTreeWidgetItem *pCurrent, QTreeWidge
     //    // Is this layer Force on?
     //    settingsEditor.SetEnabled(pLayerItem->pLayer->bActive);
 
-    ui_->lineEditDesc->setText(pLayerItem->layer_file->description);
+    //ui_->lineEditDesc->setText(pLayerItem->layer_file->description);
```

Comment:
Why are these lines commented out?

Reply:
The description UI field doesn't exist anymore as it was not really implemented...

This was used on multiple instances, remove both of them...